service/src/main/java/skills/services/admin/UserAchievementExpirationService.groovy

```diff
@@ -75,7 +75,7 @@ class UserAchievementExpirationService {
             }
         } else if (expirationAttrs.expirationType == ExpirationAttrs.DAILY) {
             LocalDateTime achievedOnOlderThan = now.minusDays(expirationAttrs.every)
-            expireAchievementsForSkillAchievedBefore(skillAttributesDef.skillRefId, achievedOnOlderThan?.toDate())
+            expireAchievementsForSkillAchievedBefore(skillAttributesDef, achievedOnOlderThan?.toDate())
```

Comment:
`UserAchievementExpirationService.groovy` can be fully rolled back; no changed needed for this PR as far as I can tell